apps/frontend/src/utils/lib/constants.ts

```diff
@@ -54,23 +54,23 @@ export const DEFAULT_NETWORK_INDEX = parseFloat(
 
 export const devnet = process.env.NODE_ENV === "development";
 
-export const stacksNetwork =
-  process.env.NODE_ENV === "production" ? StacksMainnet : StacksTestnet;
+// export const stacksNetwork =
+//   process.env.NODE_ENV === "production" ? StacksMainnet : StacksTestnet;
 
 export const TARAL_IMPORTER_CONTRACT = devnet
   ? "ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.taral-importer"
   : "ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.taral-importer";
 
 export const TARAL_BANK_CONTRACT = devnet
-  ? "ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.taral-bank"
+  ? "ST3PF13W7Z0RRM42A8VZRVFQ75SV1K26RXEP8YGKJ.taral-bank-v1"
   : process.env.NEXT_PUBLIC_TARAL_BANK_CONTRACT || "";
 
 export const SUSDT_CONTRACT = devnet
-  ? "ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.token-susdt"
+  ? "STRP7MYBHSMFH5EGN3HGX6KNQ7QBHVTBPF1669DW.susdt-token-v2"
```

Comment:
Will this be customized depenfing on the chosen network?

Reply:
@dorucioclea right now if we are deploying to mainnet we need to change the env variable value, cause other wise even in mainnet network it will point to testnet env value , this one I actually wanted to discuss with you so this PR makes sure the contract calls are based on in app network. So to make it dynamically choose contract addresses we could add more env variables like `TARAL_BANK_TESTNET_CONTRACT`
, `TARAL_BANK_MAINNET_CONTRACT`. So that i can initialise the contract address value with this env variable. What do you say?